src/__test__/actions/users.action.test.js

```diff
@@ -88,4 +92,20 @@ describe('Asset Action tests', () => {
       });
     });
   });
+
+  it('should dispatch CREATE_SECURITY_USER_SUCCESS when addSecurityUser is called successfully', () => {
+    mock.onPost(url3).reply(201, SecurityUser);
+    return store.dispatch(addSecurityUser()).then(() => {
+      expect(store.getActions()[0].type).toEqual(CREATE_SECURITY_USER_SUCCESS);
+      expect(store.getActions()[1].type).toEqual(UPDATE_TOAST_MESSAGE_CONTENT);
+    });
+  });
+
+  it('should dispatch CREATE_SECURITY_USER_SUCCESS when addSecurityUser is called successfully', () => {
+    mock.onPost(url3).reply(401);
+    return store.dispatch(addSecurityUser()).then(() => {
```

Comment:
The two comments above about **_"toContainEqual"_** and _**"Single Responsibility"**_ principle applies here